backend/metadata/databases/default/tables/core_suggestions.yaml

```diff
@@ -0,0 +1,70 @@
+table:
+  schema: core
+  name: suggestions
+object_relationships:
+  - name: section
+    using:
+      manual_configuration:
+        remote_table:
+          schema: core
+          name: sections
+        insertion_order: null
+        column_mapping:
+          section_id: id
+insert_permissions:
+  - role: authenticated
+    permission:
+      check:
+        section:
+          agreement:
+            group:
+              group_users:
+                user_id:
+                  _eq: X-Hasura-User-Id
+      columns:
+        - id
+        - created_at
+        - updated_at
+        - section_id
+        - content
+select_permissions:
+  - role: authenticated
+    permission:
+      columns:
+        - id
+        - section_id
+        - content
+        - created_at
+        - updated_at
+      filter:
+        section:
+          agreement:
+            group:
+              group_users:
+                user_id:
+                  _eq: X-Hasura-User-Id
+update_permissions:
+  - role: authenticated
+    permission:
+      columns:
+        - content
+        - updated_at
+      filter:
+        section:
+          agreement:
+            group:
+              group_users:
+                user_id:
+                  _eq: X-Hasura-User-Id
+      check: null
+delete_permissions:
```

Comment:
Let's remove the option do delete a suggestion for the time being (maybe in the future we'll add user_id column to the table like you suggested, and then allow deletion only if the requesting user is the author of the suggestion).